lumigator/frontend/src/components/experiments/LExperimentForm.vue

```diff
@@ -231,30 +231,50 @@ async function handleRunExperimentClicked() {
     system_prompt: experimentPrompt.value || defaultPrompt.value,
   }
 
-  const workflows = await experimentsService.createExperimentWithWorkflows(
+  const [experimentId, workflowResults] = await experimentsService.createExperimentWithWorkflows(
     experimentPayload,
     modelSelection.value.selectedModels,
   )
+  // Separate successful workflows and errors
+  const workflows = workflowResults.filter((r) => r.status === 'fulfilled').map((r) => r.value)
```

Comment:
nit: `r.status` can be misinterpreted as the workflow status instead of the promise's

```suggestion
  const successfulResponses = createWorkflowResponses.filter((promise) => promise.status === 'fulfilled').map((promise) => promise.value)
```